torobo/README.md

```diff
@@ -0,0 +1,16 @@
+# Torobo Description (MJCF)
+
+Requires MuJoCo 3.1.0 or later.
+
+## Overview
```

Comment:
Can you add the steps you took to derive the XML? Was this XML generated from a CAD model or from a URDF? In general, we strive to have a detailed reproduction paragraph, e.g. see [here](https://github.com/google-deepmind/mujoco_menagerie/tree/main/anybotics_anymal_b#urdf--mjcf-derivation-steps)